### DISCUSSION TITLE
Is it possible to add the popular "Add to updateset" UI Action using the SN Utils?

### DISCUSSION THREAD

**Discussion by RahmanM** [arnoudkooi/SN-Utils#521] [Ideas]
Hi there,

This is most probably a question than an issue.

Every new instance that I develope in I find my self adding the popular UI Action "Add to updateset". The UI Action that enables to add any record to an update-set.

Just wondering if is it possible for UN Utils to mimic the same functionality so everyone does not have to create that UI action manually?

Many thanks

**Comment by arnoudkooi** [comment]
That is an awesome tool indeed, it has a lot of complexity in it around grouping and scoping that is not feasible to mimic in SN utils.

Ill convert this to an idea discussion ad leave it open for a bit

**Comment by avroguha** [comment]
This is something that will help for sure, today we are using update set to add to update set. We primarily use this for objects that doesn't get added to update set OOTB.
